src/applications/disability-benefits/all-claims/analytics-functions.js

```diff
@@ -2,8 +2,10 @@ import { recordEventOnce } from 'platform/monitoring/record-event';
 import get from 'platform/utilities/data/get';
 import { HOMELESSNESS_TYPES } from './constants';
 
-const objectIsEmpty = (path, formData) =>
-  Object.values(get(path, formData, {})).every(option => !option);
+const objectIsEmpty = (obj, path) =>
+  Object.values(typeof path === 'string' ? get(path, obj, {}) : obj).every(
```

Comment:
[This line](https://github.com/department-of-veterans-affairs/vets-website/pull/10185/files#diff-fe03f9337966fbba2ab7cf04432762a0R55) is calling it without the `path`, which triggered [this error](http://sentry.vetsgov-internal/vets-gov/website-production/issues/60824/?query=is%3Aunresolved).